templates/kp_extraction/kp_extraction_emb.html

```diff
@@ -46,44 +55,81 @@
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
     </div>
 
     <div class='row' id="setpopup">
-            <div data-role="main" class="ui-content">
-        <a href="#control_pannel" data-rel="popup" class="ui-btn ui-btn-inline ui-corner-all ui-icon-gear ui-btn-icon-left">Settings</a>
-
-        <div data-role="popup" id="control_pannel" class="ui-content" style="min-width:250px;">
-            <div>
-              <h3>Settings</h3>
-              <label for="nbkp">Number of keyphrases</label>
-              <input type="number" name="nbkp" id="nbkp" value=10>
+        <div data-role="main" class="ui-content">
+            <a href="#control_pannel" data-rel="popup" class="ui-btn ui-btn-inline ui-corner-all ui-icon-gear ui-btn-icon-left">Settings</a>
+
+            <div data-role="popup" id="control_pannel" class="ui-content" style="min-width:250px;">
+                <div>
+                    <h3>Settings</h3>
+                    <label for="nbkp">Number of keyphrases</label>
+                    <input type="number" name="nbkp" id="nbkp" value=10>
+                </div>
             </div>
         </div>
-      </div>
     </div>
 
+    <!-- Footer -->
+    <div class="footer-outer">
+        <div class="footer footer--compact footer--light footer-inner">
+            <a class="logo">
+                <span class="logo-lifeform" aria-label="Swisscom Logo"></span>
+            </a>
+            <ul class="nav-items">
+                <li class="nav-item">
+                    <a href="https://swisscom.ch/ai" class="nav-link" target="_blank">
+                        AI for your business?
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </div>
 
-
-
-
-
-
-
-      
     <!-- jQuery first, then Tether, then Bootstrap JS. -->
-    <script  src="https://code.jquery.com/jquery-3.1.1.min.js"   integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="   crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb"
+        crossorigin="anonymous"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.matchHeight/0.7.0/jquery.matchHeight-min.js"></script>
     <script type=text/javascript src="{{ url_for('static', filename='javascripts/common.js') }}"></script>
+    <!--
```

Comment:
Commenting out the old stuff is fine while working on the feature, but when you commit, it's important to remove the old code and delete the files that are not used anymore